openslides/motions/config_variables.py

```diff
@@ -0,0 +1,179 @@
+from django.core.validators import MinValueValidator
+from django.utils.translation import ugettext as _
+from django.utils.translation import pgettext, ugettext_lazy
+
+from openslides.core.config import ConfigVariable
+from openslides.poll.models import PERCENT_BASE_CHOICES
+
+from .models import Workflow
+
+
+def get_workflows():
+    return [{'value': str(workflow.pk), 'display_name': ugettext_lazy(workflow.name)}
```

Comment:
Rename to `get_workflow_choices`.

Docstring: `Returns a list of all workflows to be used as choices for the config variable 'motions_workflow'. Each list item contains the pk and the display name.`